lib/ipmeta.c

```diff
@@ -409,14 +469,18 @@ void ipmeta_dump_record(ipmeta_record_t *record, char *ip_str, int num_ips)
 
 #define PRINT_RECORD_HEADER(function, file)                                    \
   do {                                                                         \
-    function(file, "ip-prefix" SEPARATOR "num-ips" SEPARATOR "id" SEPARATOR    \
-                   "country-code" SEPARATOR "continent-code" SEPARATOR         \
-                   "region" SEPARATOR "city" SEPARATOR "post-code" SEPARATOR   \
-                   "latitude" SEPARATOR "longitude" SEPARATOR                  \
-                   "metro-code" SEPARATOR "area-code" SEPARATOR                \
-                   "region-code" SEPARATOR "connection-speed" SEPARATOR        \
-                   "polygon-ids" SEPARATOR "asn" SEPARATOR "asn-ip-cnt"        \
-                   "\n");                                                      \
+    function(file,                                                             \
+             "ip-prefix" SEPARATOR "num-ips" SEPARATOR "id" SEPARATOR          \
+             "country-code" SEPARATOR "continent-code" SEPARATOR               \
+             "region" SEPARATOR "city" SEPARATOR "post-code" SEPARATOR         \
+             "latitude" SEPARATOR "longitude" SEPARATOR "metro-code" SEPARATOR \
+             "area-code" SEPARATOR "region-code" SEPARATOR                     \
+             "connection-speed" SEPARATOR "polygon-ids" SEPARATOR              \
+             "asn" SEPARATOR "asn-ip-cnt" SEPARATOR "local_code" SEPARATOR     \
```

Comment:
please replace the `_` with `-` to be consistent with other field names.
also, isn't it "locale" rather than "local"?